packages/yew/src/virtual_dom/vcomp.rs

```diff
@@ -283,6 +308,22 @@ impl<COMP: BaseComponent> fmt::Debug for VChild<COMP> {
     }
 }
 
+#[cfg(feature = "ssr")]
+mod feat_ssr {
+    use super::*;
+
+    impl VComp {
+        pub(crate) async fn render_to_string(&self, w: &mut String, parent_scope: &AnyScope) {
```

Comment:
Why not just add `render_to_string` to already already existing `impl VComp`?

```rust
impl VComp {
    // other functions

    #[cfg(feature = "ssr")]
    pub(crate) async fn render_to_string(&self, w: &mut String, parent_scope: &AnyScope) { ... }
}
```
Same for other `mod feat_ssr`s (where applicable)

Reply:
This is to minimise the number of feature flags required to gate `ssr`.

I find it difficult to get the feature flags right if it was not isolated into a mod as you will need 1 for each `use`, method, etc. You will also need a `#[doc(cfg(...))]` for each `#[cfg(...)]` if you want docs to show. Doing it on the mod level then you will only need to flag it once.

Reply:
That's understandable. When we have multiple functions, this way scales well